ci/ci_test.sh

```diff
@@ -17,6 +17,13 @@ elif [[ $SELECTED_TEST_SUITE == $"all" ]]; then
   rake test:plugins         # Run all plugins tests
 else
   echo "Running core tests"
-  rake test:install-core    # Install core dependencies for testing.
+  rake plugin:install-core    # Install core dependencies for testing.
```

Comment:
no, no before merging we move back to the original. This has been made just for the idea of making the test face easier.